site/docs/guide/files.md

```diff
@@ -5,107 +5,112 @@ next: ./games.md
 
-  const file = await ctx.getFile(); // valid for 1 hour
+  const file = await ctx.getFile(); // valid for at least 1 hour
   const path = file.file_path; // file path on Bot API server
   await ctx.reply("Download your own file again: " + path);
 });
 ```
 
-::: tip Passing file_id to getFile
-On context object `getFile` is [a shortcut](/guide/context.md#shortcuts) that will get you a file from current message.
-If you want to get another file while handling a message - use `ctx.api.getFile(file_id)`.
+::: tip Passing a Custom file_id to getFile
+On the context object, `getFile` is [a shortcut](/guide/context.md#shortcuts), and will fetch information for a file on the current message.
+If you want to get a different file while handling a message, use `ctx.api.getFile(file_id)` instead.
 :::
 
 > Check out [the `:media` and `:file` shortcuts](/guide/filter-queries.md#shortcuts) for filter queries if you want to receive any kind of file.
 
-You can use the file path to know where on the Telegram Bot API servers your file resides.
-You can then download it again using the URL `https://api.telegram.org/file/bot<token>/<file_path>` where `<token>` must be replaced by your bot token, and `<file_path>` must be replaced by the file path.
+Once you have called `getFile`, you can use the returned `file_path` to download the file using this URL `https://api.telegram.org/file/bot<token>/<file_path>`, where `<token>` must be replaced by your bot token.
 
 ::: tip Files Plugin
-grammY does not ship its own file downloader, but you can install it using [the official files plugin](/plugins/files.md).
-It allows you to download files via `await file.download()`, and to obtain their URL via `file.getUrl()`.
+grammY does not come bundled with its own file downloader, but you can install [the official files plugin](/plugins/files.md).
+This allows you to download files via `await file.download()`, and to obtain a constructed download URL for them via `file.getUrl()`.
 :::
 
 ## Sending Files
 
 Telegram bots have [three ways](https://core.telegram.org/bots/api#sending-files) to send files:
 
-1. Via `file_id`, i.e. by sending a file by identifier that already exists in another chat.
-2. Via URL, i.e. by passing a public file URL to Telegram that will in turn download and send the file.
+1. Via `file_id`, i.e. by sending a file by an identifier that is already known to the bot.
+2. Via URL, i.e. by passing a public file URL, which Telegram downloads and sends for you.
 3. Via uploading your own file.
 
 ### Via `file_id` or URL
 
-The first two means are simple: you just pass the respective value as `string` and you're done.
+The first two methods are simple: you just pass the respective value as a `string`, and you're done.
 
 ```ts
-// Sending by file_id
+// Sending via file_id
 await ctx.replyWithPhoto(existingFileId);
 
-// Sending by URL
+// Sending via URL
 await ctx.replyWithPhoto("https://avatars.githubusercontent.com/u/81446018");
```

Comment:
By the way @KnorpelSenf, since the newer examples you added below use `https://grammy.dev/Y.png`, do we want to change this also?

```suggestion
await ctx.replyWithPhoto("https://grammy.dev/Y.png");
```

Reply:
Yes! Good idea